tests/unit/test_runner.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+from unittest.mock import patch
+
+
+@patch("shutil.chown")
+def test_download(chown, runner):
+    runner.download()
```

Comment:
Technically, this isn't a unit test since it performs an actual download, meaning this could fail depending due to network or disk issues.

Reply:
I agree, I started with this for faster testing, and then checked that the download worked in the integration testing. Do you think it's best to put a pytest.skip on it or just remove it? It's a lot easier to test the downloading locally and I just never went back and removed it. It does seem helpful so I'm inclined to put it behind a flag or something.